sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 ## 1.1.0-beta.5 (Unreleased)
 
+### New Features
+- Added option to write to shared token cache from `ClientSecretCredential`, `ClientCertificateCredential.
+- Added new Developer credentials `IntelliJCredential`, `VsCodeCredential` and `AzureCliCredential`.
+- First preview of new API for authenticating users with `DeviceCodeCredential` and `InteractiveBrowserCredential`
+- Added method `authenticate` which pro-actively interacts with the user to authenticate if necessary and returns a serializable `AuthenticationRecord`
```

Comment:
Was this new method added to a top level interface or to specific classes?

Reply:
Added a tab to this text, it is supposed to be under the previous comment.